### DISCUSSION TITLE
button-card / welcome scenes

### DISCUSSION THREAD

**Discussion by fabiorcosta** [UI-Lovelace-Minimalist/UI#951] [Q&A] [answered]
I have a shortcut group with 2 scenes. But as per image, the font is too small and is wraping. Is there a way to auto-break or increase it?

![image](https://user-images.githubusercontent.com/82267134/185679611-e1272a87-eab0-4b97-9d57-530b2fe34b5e.png)

**Comment by basbruss** [comment, marked as answer]
Will be fixed in the next release with this pr https://github.com/UI-Lovelace-Minimalist/UI/pull/933
